src/LinuxTracing/testdata/target.cc

```diff
@@ -0,0 +1,116 @@
+// g++ target.cc -O0 -fno-omit-frame-pointer -momit-leaf-frame-pointer -o target_fp
+// g++ target.cc -O0 -fomit-frame-pointer -o target_no_fp
+
+#include <sched.h>
+
+#include <cerrno>
+#include <cstdint>
+#include <cstdio>
+#include <cstring>
+#include <ctime>
+#include <vector>
+
+inline uint64_t timestamp_ns() {
```

Comment:
I guess I don't want you to spend time changing all the addresses again, but I think this file should have been reduced to only what you need for the purposes of the tests, i.e., two simple functions...

Reply:
My thought was that if you remove everything that's not needed, the result is so different from the original that you should rebuild (otherwise it gets very confusing), but that changes all the addresses, and in the interest of not wasting your time then you should leave it as it is.
But let's just say that I would do it differently another time: this binary is overcomplicated for the tests it's used in and does a lot of weird things (`CPU_SET`, `stack_filler`, print moving average of time, ...).